SlackAPI/AttachmentActionOption.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace SlackAPI
+{
+    //See: https://api.slack.com/docs/message-menus
```

Comment:
This should be a proper XML document so people using the library can see what to reference